src/http/http_1.rs

```diff
@@ -8,39 +8,29 @@ use crate::cli::CLIError;
 
 pub async fn start_http_1(sc: Arc<ServerConfig>) -> std::prelude::v1::Result<(), anyhow::Error> {
   let addr = sc.addr();
-  Ok(
-    sc.tokio_runtime()?
-      .spawn(async move {
-        let sc_cloned = sc.clone();
+  let make_svc_single_req = make_service_fn(|_conn| {
+    let state = Arc::clone(&sc);
+    async move {
+      Ok::<_, anyhow::Error>(service_fn(move |req| {
+        handle_single_request(req, state.server_context.clone())
+      }))
+    }
+  });
 
-        let make_svc_single_req = make_service_fn(|_conn| {
-          let state = Arc::clone(&sc_cloned);
-          async move {
-            Ok::<_, anyhow::Error>(service_fn(move |req| {
-              handle_single_request(req, state.server_context.clone())
-            }))
-          }
-        });
-
-        let make_svc_batch_req = make_service_fn(|_conn| {
-          let state = Arc::clone(&sc_cloned);
-          async move {
-            Ok::<_, anyhow::Error>(service_fn(move |req| {
-              handle_batch_request(req, state.server_context.clone())
-            }))
-          }
-        });
-        let builder = hyper::Server::try_bind(&addr).map_err(CLIError::from)?;
-
-        let server: std::prelude::v1::Result<(), hyper::Error> = if sc_cloned.blueprint.server.enable_batch_requests {
-          builder.serve(make_svc_batch_req).await
-        } else {
-          builder.serve(make_svc_single_req).await
-        };
-
-        log_launch(sc.as_ref());
-        server.map_err(CLIError::from)
-      })
-      .await??,
-  )
+  let make_svc_batch_req = make_service_fn(|_conn| {
+    let state = Arc::clone(&sc);
+    async move {
+      Ok::<_, anyhow::Error>(service_fn(move |req| {
+        handle_batch_request(req, state.server_context.clone())
+      }))
+    }
+  });
+  let builder = hyper::Server::try_bind(&addr).map_err(CLIError::from)?;
+  log_launch(sc.as_ref());
```

Comment:
On `log_launch`, can we check for the used protocol also? The current implementation uses `https://{addr}` even for HTTP 1

Reply:
Ya let me fix that

Reply:
![image](https://github.com/tailcallhq/tailcall/assets/100464352/ccaf7c9e-fdec-4675-a431-6eb9430a7403)

The expected output here would be 
```
[2023-11-21T16:56:53Z INFO  tailcall::http] 🌍 Playground: http://0.0.0.0:8000
```
since we're in a non-HTTPS server